Neos.Neos/Resources/Private/Templates/FusionObjects/BreadcrumbMenu.html

```diff
@@ -9,7 +9,7 @@
 					{item.label}
 				</f:then>
 				<f:else>
-					<neos:link.node node="{item.node}" />
+					<f:link.external uri="{item.uri}" title="{item.label}">{item.label}</f:link.external>
```

Comment:
Feels a bit weird to see `external` in there… Feels like that conveys additional meaning, adding `target="_blank" or doing some other fancy trick.

Put differently: Would a "novice integrator" use `link.external` to link to URIs generated for nodes?